### PR TITLE
Docs - Minor edits Applying Style Guide and Diataxis

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* Added meta tags for all docs.
* Updated package.json and package-lock.json
* Updated test-docs-build.yml action
* Edited all docs.
* Removed yarn.lock as it conflicts with package-lock.json

## Type of change

- Enhancement (a non-breaking change that adds functionality)
- Documentation Enhancement

## How the change has been tested

Wiki was built locally.  

## Change checklist

- [x] I have followed the contribution guidelines for this project

### CODE REVIEW

**Comment by lucas-tortora** [comment]
@iotaledger/tech-writers could you please review this?

File: documentation/docs/config_reference.md [hunk 1]
```diff
@@ -1,86 +1,131 @@
...
-See [KeyspaceConfig](chronicle-storage/src/config.rs#KeyspaceConfig)
+### keyspaces: Vec&lt;KeyspaceConfig>
 
-Multiple keyspaces can be configured in order to filter incoming messages. If the `filter` feature is not used, *only the first configured keyspace will be considered* or the default (`chronicle`) if none is provided.
+You can configure multiple keyspaces to filter incoming messages. If you are not using the `filter` feature, *only the
+first configured keyspace will be considered* or the default (`chronicle`) if none is provided.
 
-In addition to the keyspace name, each requires a map of datacenters (name -> replication factor). See [here](https://university.scylladb.com/courses/scylla-essentials-overview/lessons/architecture/topic/datacenter/) for more information about datacenters in ScyllaDB.
+In addition to the keyspace name, each keyspace requires a map of datacenters (name -> replication factor). You can find
+more configuration about datacenters in ScyllaDB in
```
Comment (charlesthompson3) [inline documentation/docs/config_reference.md:22]:
```suggestion
more configurations about datacenters in ScyllaDB in
```

File: documentation/docs/config_reference.md [hunk 2]
```diff
@@ -1,86 +1,131 @@
...
+### max_log_size: Option&lt;u64>
 
-### `max_log_size: Option<u64>`
 The upper limit of the log_file_size.
 
-NOTE: Ensure to use a limit within your filesystem range.
\ No newline at end of file
+:::note
+
+Ensure to use a limit within your filesystem range.
```
Comment (charlesthompson3) [inline documentation/docs/config_reference.md:128]:
```suggestion
Ensure you use a limit within your filesystem range.
```

File: documentation/docs/contribute.md [hunk 3]
```diff
@@ -1,3 +1,35 @@
...
+Chronicle.rs is an open source project hosted on [GitHub](https://github.com/iotaledger/chronicle.rs).
+You can contribute by:
+
+* [Reporting a bug](https://github.com/iotaledger/chronicle.rs/issues/new?assignees=&labels=bug&template=bug_report.md&title=).
+* [Requesting a feature](https://github.com/iotaledger/chronicle.rs/issues/new?assignees=&labels=enhancement&template=enhancement.md&title=)
+* [Creating a pull request](https://github.com/iotaledger/chronicle.rs/pulls)
+
+:::info
+
+Please read make sure you have read our [contributing guidelines](https://github.com/iotaledger/documentation/blob/develop/.github/CONTRIBUTING.md) before contributing.
```
Comment (charlesthompson3) [inline documentation/docs/contribute.md:25]:
```suggestion
Please make sure you have read our [contributing guidelines](https://github.com/iotaledger/documentation/blob/develop/.github/CONTRIBUTING.md) before contributing.
```

File: documentation/docs/getting_started.md [hunk 4]
```diff
@@ -1,79 +1,117 @@
+---
+image: /img/chronicle_icon.png 
+description:  Get started with Chronicle. Prerequisites, installation and build instructions to run your own Chronicle permanode.
```
Comment (charlesthompson3) [inline documentation/docs/getting_started.md:3]:
```suggestion
description: Get started with Chronicle. Prerequisites, installation, and build instructions to run your own Chronicle permanode. 
```

File: documentation/docs/getting_started.md [hunk 5]
```diff
@@ -1,79 +1,117 @@
...
 ## Installation
 
-Either download the provided executable (you should only do this if you do not wish to use the filtering functionality), or build it yourself.
+If you do not wish to use the filtering functionality, you can download the Chronicle executable.
+Alternatively, you can [build Chronicle](#build-chronicle) yourself. 
+
+### Build Chronicle
 
-### Building Chronicle
+You can follow these steps to build Chronicle from source:
```
Comment (charlesthompson3) [inline documentation/docs/getting_started.md:71]:
```suggestion
You can follow these steps to build Chronicle from the source:
```

File: documentation/docs/welcome.md [hunk 6]
```diff
@@ -1,11 +1,21 @@
+---
+image: /img/chronicle_icon.png
+description: Chronicle provides tools for managing and accessing permanode solutions using the IOTA actor framework backstage to allow you to store, explore and filer IOTA messages.
```
Comment (charlesthompson3) [inline documentation/docs/welcome.md:3]:
```suggestion
description: Chronicle provides tools for managing and accessing permanode solutions using the IOTA actor framework backstage to allow you to store, explore, and filer IOTA messages.
```

File: documentation/docs/welcome.md [hunk 7]
```diff
@@ -17,11 +27,25 @@ This is alpha software, so there may be performance and stability issues. Please
...
+
+The entry point for the Chronicle application.
+
+### [API](https://github.com/iotaledger/chronicle.rs/tree/main/chronicle-api)
+
+API that allows you to access stored messages.
+
+### [Broker](https://github.com/iotaledger/chronicle.rs/tree/main/chronicle-broker)
+
+Allow you to subscribe to incoming messages from IOTA nodes.
```
Comment (charlesthompson3) [inline documentation/docs/welcome.md:40]:
```suggestion
Allows you to subscribe to incoming messages from IOTA nodes.
```

**Review by charlesthompson3 — APPROVED** [review]
👌

**Pull request closed without merging**